data/yam/autoyast/support_images/sles15sp3_install_gnome_default_patterns_aarch64.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <bootloader t="map">
+    <global t="map">
+      <timeout t="integer">-1</timeout>
+    </global>
+    <loader_type>grub2-efi</loader_type>
+  </bootloader>
+  <firewall t="map">
+    <enable_firewall t="boolean">true</enable_firewall>
+    <start_firewall t="boolean">true</start_firewall>
+  </firewall>
+  <general t="map">
+    <mode t="map">
+      <confirm t="boolean">false</confirm>
+    </mode>
+  </general>
+  <keyboard t="map">
+    <keyboard_values t="map">
+      <delay/>
+      <discaps t="boolean">false</discaps>
+      <numlock>bios</numlock>
+      <rate/>
+    </keyboard_values>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language t="map">
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <networking t="map">
+    <interfaces t="list">
+      <interface t="map">
+        <bootproto>dhcp</bootproto>
+        <device>eth0</device>
+        <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+    <keep_install_network t="boolean">true</keep_install_network>
+  </networking>
+  <ntp-client t="map">
+    <ntp_policy>auto</ntp_policy>
+  </ntp-client>
+  <partitioning t="list">
```

Comment:
Will you follow the https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17594/files to set partition without home section? It will create home section by default, but this will cause disk space issue for migration test. So for the other profile files.

Reply:
I would suggest using sle15spx autoyast configuration files :)

Reply:
Done

Reply:
I started from this copy:
```
cp data/yam/autoyast/support_images/sles15sp5_install_gnome_default_patterns_aarch64.xml      data/yam/autoyast/support_images/sles15sp3_install_gnome_default_patterns_aarch64.xml
cp data/yam/autoyast/support_images/sles15sp5_install_gnome_default_patterns_s390x.xml        data/yam/autoyast/support_images/sles15sp3_install_gnome_default_patterns_s390x.xml
cp data/yam/autoyast/support_images/sles15sp5_install_gnome_default_patterns_x86_64.xml       data/yam/autoyast/support_images/sles15sp3_install_gnome_default_patterns_x86_64.xml
cp data/yam/autoyast/support_images/sles15sp5_install_textmode_default_patterns_aarch64.xml   data/yam/autoyast/support_images/sles15sp3_install_textmode_default_patterns_aarch64.xml
cp data/yam/autoyast/support_images/sles15sp5_install_textmode_default_patterns_s390x.xml     data/yam/autoyast/support_images/sles15sp3_install_textmode_default_patterns_s390x.xml
cp data/yam/autoyast/support_images/sles15sp5_install_textmode_default_patterns_x86_64.xml    data/yam/autoyast/support_images/sles15sp3_install_textmode_default_patterns_x86_64.xml
```

Reply:
I agree with Richard, it is better to use SLE15 autoyast configuration for partition section to avoid potential risk.

Reply:
Added _partition_ from _sles15sp5_:
```xml
  <partitioning t="list">
    <drive t="map">
      <initialize t="boolean">true</initialize>
      <use>all</use>
    </drive>
  </partitioning>
```